assets/js/map.js

```diff
@@ -108,7 +99,7 @@
   function renderMapLocations(locations) {
     // size accessor (keeping things DRY)
     var size = function getSize(d) {
```

Comment:
JavaScript is, um, :sparkles: fun :sparkles: because you can define methods two ways:

```javascript
var getSize = function(d) {
  // ...
};

// or

function getSize(d) {
  // ...
}
```

Shouldn't need the name on the declaration side *and* the assignment side.